examples/heartbeat_demo/alicia.py

```diff
@@ -0,0 +1,142 @@
+from nucypher.characters.lawful import Bob, Ursula
+from nucypher.config.characters import AliceConfiguration
+from nucypher.config.storages import LocalFileBasedNodeStorage
+from nucypher.crypto.powers import EncryptingPower, SigningPower
+from nucypher.network.middleware import RestMiddleware
+from nucypher.utilities.logging import simpleObserver
+from umbral.keys import UmbralPublicKey
+
+from doctor_keys import get_doctor_pubkeys
+
+import datetime
+import os
+import shutil
+import maya
+import json
+from twisted.logger import globalLogPublisher
+
+
+POLICY_FILENAME = "policy.json"
+
+
+######################
+# Boring setup stuff #
+######################
+#
+# # Twisted Logger
+globalLogPublisher.addObserver(simpleObserver)
+#
+# # Temporary file storage
+TEMP_ALICE_DIR = "{}/alicia-files".format(os.path.dirname(os.path.abspath(__file__)))
+TEMP_URSULA_CERTIFICATE_DIR = "{}/ursula-certs".format(TEMP_ALICE_DIR)
```

Comment:
This can be setup deprecated with PR #562 since certificate storage is automatically handles by `NodeStorage`s.

Reply:
So, should we merge this example after #562 then?

Reply:
@KPrasch: What's the story here at this point?

Reply:
So how shall we handle this?  Merge with a mind to fix later when we do the big one?

Reply:
Yes, I think so.

Reply:
OK, in that case (and given @KPrasch's comment below vis a vis #599), I'm :+1: for merge.